core/meson.build

```diff
@@ -0,0 +1,6 @@
+core_sources = ['src/math/matrix.cpp', 'src/math/vector.cpp']
```

Comment:
Are we going to have to update this line every time we add a new `.cpp` file to the library? Do you know if there's a way to do this automatically?

Reply:
Yes, that's the preferred pattern. It's explained here: https://mesonbuild.com/FAQ.html#why-cant-i-specify-target-files-with-a-wildcard